### DISCUSSION TITLE
Change font size per tab

### DISCUSSION THREAD

**Discussion by mihailigoshin** [kovidgoyal/kitty#7651] [Q&A] [answered]
Hi guys, 

Is there a way to change the font size per tab not per window?

**Comment by kovidgoyal** [comment, marked as answer]
On Sat, Jul 20, 2024 at 05:06:18AM -0700, Mihail Igoshin wrote:
> Hi guys,
>
> Is there a way to change the font size per tab not per window?

No, there isnt. Every OS Window can have only a single font size.
